app/assets/javascripts/discourse/app/templates/components/categories-boxes-with-topics.hbs

```diff
@@ -1,37 +1,39 @@
-{{#each categories as |c|}}
-  <div data-notification-level={{c.notificationLevelString}} style={{unless noCategoryStyle (html-safe (concat (border-color c.color) (category-color-variable c.color)))}} class="category category-box category-box-{{c.slug}} {{if c.isMuted "muted"}} {{if noCategoryStyle "no-category-boxes-style"}}">
-    <div class="category-box-inner">
-      <div class="category-box-heading">
-        <a href={{c.url}}>
-          {{#unless c.isMuted}}
-            {{#if c.uploaded_logo.url}}
-              <CdnImg @src={{c.uploaded_logo.url}} @class="logo" @width={{c.uploaded_logo.width}} @height={{c.uploaded_logo.height}} @alt="" />
-            {{/if}}
-          {{/unless}}
-
-          <h3>
-            <CategoryTitleBefore @category={{c}} />
-            {{#if c.read_restricted}}
-              {{d-icon lockIcon}}
-            {{/if}}
-            {{c.name}}
-          </h3>
-        </a>
-      </div>
+<section class="category-boxes-with-topics {{if this.anyLogos "with-logos" "no-logos"}}" ...attributes>
```

Comment:
Not really that important, but why do we use a `<section>` here instead of a div?

Reply:
It comes from here https://github.com/discourse/discourse/blob/c59f1729a6516ffa7478fbb74336da5b7159fd9c/app/assets/javascripts/discourse/app/components/categories-boxes-with-topics.js#L7